test/sqllogictest/privilege_grants.slt

```diff
@@ -3540,22 +3540,22 @@ COMPLETE 0
 ## Tables
 
 query B
-SELECT has_table_privilege('t3', 'SELECT')
+SELECT has_table_privilege('d1.s1.t3', 'SELECT')
```

Comment:
The test passes only if you have `d1.s1` in your search path, otherwise it errors because there is no `t3` in `materialize.public`. I can work with the cloud team to check if there are any deps on the privilege functions before we merge this. A good reminder to not ship buggy code to prod.